Content.Shared/Stacks/StackThresholdComponent.cs

```diff
@@ -0,0 +1,13 @@
+namespace Content.Shared.Stacks;
+
+[RegisterComponent]
+public sealed partial class StackLayerThresholdComponent : Component
+{
+    /// <summary>
+    /// A list of thresholds to check against the number of things in the stack.
+    /// Each exceeded threshold will cause the next layer to be displayed.
+    /// Should be sorted in ascending order.
+    /// </summary>
+    [DataField(required: true)]
+    public List<int> Thresholds = new List<int>();
```

Comment:
You can just use a collection expression here.

```suggestion
    public List<int> Thresholds = [];
```

Reply:
I personally think using `new()` is more explicit and better than collection expressions.

Reply:
Sure, but the `List<int>` part is redundant. You can just do `new ();`

Either way works.